steps/firebase-change-file-by-env/0.0.1/step.yml

```diff
@@ -0,0 +1,83 @@
+title: Firebase Change File by environment
+summary: |
+  Change file change firebase file according to branch
+description: |
+  Change file between firebase project according to branch in your repository
+  * Important Suggestion *: You must create a folder in the root of your project called 'firebase_env'
+   and inside have two folders for production and development, the step suggests in its variables
+   and the corresponding folders.
+website: https://github.com/manuelabarca/bitrise-step-firebase-change-file-by-env
+source_code_url: https://github.com/manuelabarca/bitrise-step-firebase-change-file-by-env
+support_url: https://github.com/manuelabarca/bitrise-step-firebase-change-file-by-env/issues
+published_at: 2021-03-31T01:05:02.040459-03:00
+source:
+  git: https://github.com/manuelabarca/bitrise-step-firebase-change-file-by-env.git
+  commit: 753198d247c8969aaf255ea08d62ac6d2a8b4fcd
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
```

Comment:
Why not other multiplatform ones like flutter?

Reply:
I forgot it, I update it